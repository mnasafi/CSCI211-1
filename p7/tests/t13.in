echo t13: Insert days of week into tree, then call breadth
insert Friday
insert Monday
insert Saturday
insert Sunday
insert Wednesday
insert Thursday
insert Tuesday
breadth
