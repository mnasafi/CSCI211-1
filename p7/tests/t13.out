t13: Insert days of week into tree, then call breadth
{Friday, Monday, Saturday, Sunday, Wednesday, Thursday, Tuesday}
